packages/superset-ui-control-utils/src/types.ts

```diff
@@ -228,34 +233,42 @@ export type SharedSectionAlias =
   | 'sqlaTimeSeries'
   | 'NVD3TimeSeries';
 
-export interface ControlItem {
+export interface OverrideSharedControlItem {
   name: SharedControlAlias;
-  config: Partial<ControlConfig>;
+  override: Partial<SharedControlConfig>;
```

Comment:
This is new API. Superset app needs to be updated to support this. 

Will file a separate PR.